app/_hub/kong-inc/aws-lambda/1.0-x.md

```diff
@@ -0,0 +1,309 @@
+    different MIME type (like `application/json`), or to use a different HTTP client.
+
+---
+
+### Sending parameters
+
+Any form parameter sent along with the request will be also sent as an
+argument to the AWS Lambda function.
+
+### Notes
+
+If you do not provide an `aws.key` and `aws.secret`, the plugin uses an IAM role inherited from the instance running Kong.
+
+First, the plugin will try ECS metadata to get the role. If no ECS metadata is available, the plugin will fall back on EC2 metadata.
+
+### Known Issues
+
+#### Use a fake upstream service
+
+When using the AWS Lambda plugin, the response will be returned by the plugin
+itself without proxying the request to any upstream service. This means that
+a Service's `host`, `port`, and `path` properties will be ignored, but must
+still be specified for the entity to be validated by Kong. The `host` property
+in particular must either be an IP address, or a hostname that gets resolved by
+your nameserver.
+
+#### Response plugins
+
+There is a known limitation in the system that prevents some response plugins
+from being executed. We are planning to remove this limitation in the future.
+
+[configuration]: /latest/configuration
+[consumer-object]: /latest/admin-api/#consumer-object
+[acl-associating]: /plugins/acl/#associating-consumers
+[faq-authentication]: /about/faq/#how-can-i-add-an-authentication-layer-on-a-microservice/api?
+
+---
+### Step-By-Step Guide
+
+#### Steps
+
+You must have access to the AWS Console as a user who is allowed to operate
+with lambda functions, and create users and roles.
+
+1. Create an Execution role in AWS.
+2. Create a user that will invoke the function via Kong and test it.
+3. Create a Service and Route in Kong, add the aws-lambda plugin linked to
+our AWS function, and execute it.
+
+#### Configure
+
+1. First, let's create an execution role called `LambdaExecutor` for our
+lambda function.
+
+    In the IAM Console, create a new Role choosing the AWS Lambda service. There
+    will be no policies as our function in this example will simply execute
+    itself, giving us back a hardcoded JSON response without accessing other
+    AWS resources.
+
+2. Now let's create a user named `KongInvoker`, used by our Kong API gateway
+to invoke the function.
+
+    In the IAM Console, create a new user. Programmatic access must be provided to the user via Access and Secret keys. Then attach existing policies directly, particularly the predefined `AWSLambdaRole`. After the user creation is confirmed, store the Access Key and Secret Key in a safe place.
+
+3. Now we need to create the lambda function itself; we will do so in
+N. Virginia Region (code `us-east-1`).
+
+    In Lambda Management, create a new function `Mylambda`. There will be no blueprint because we are going to paste the code below (which is an example code snippet). Let's choose an existing role for the execution role, specifically `LambdaExecutor` created previously.
```

Comment:
in same cases in this document, this is written as `Mylambda`, and in others, `MyLambda`.

Reply:
> in same cases in this document, this is written as `Mylambda`, and in others, `MyLambda`.

@p0pr0ck5 Great catch! Done in commit 7efda68. Thanks again for your review.